StanfordHCI/foundry
# Handoff delayed fix

Handoffs were getting messed up when events were delayed. Fixes issue #125 

To test: 
Create a team, create handoffs between events. Start the team, start different types of events. When delayed, observe if the handoff is distorted.
